types/schema.d.ts

```diff
@@ -2,6 +2,13 @@ import type { UUID } from 'crypto';
 
 export type Season = 'SPRING' | 'SUMMER' | 'FALL' | 'WINTER';
 
+export interface Profile {
+  state: string;
+  email: string;
+  phone_num: string;
```

Comment:
quick note: we may remove phone_num in the future (gonna discuss w rob on fri), but we can leave it for now.